src/Microsoft.ML.Transforms/CategoryImputer.cs

```diff
@@ -0,0 +1,1829 @@
+﻿using System;
```

Comment:
(nit) copyright on all code files.

Reply:
copyright has been added. Thanks for the reminder.